src/yoroi-wallets/cardano/types.ts

```diff
@@ -78,6 +79,8 @@ export interface WalletInterface {
 
   storage: typeof storage
 
+  utxoService: UtxoService | null
```

Comment:
hoping for `UtxoService`

Reply:
instantiate in constructor, and get rid of `null`